### PR TITLE
Remove count of available convertors

### PR DESCRIPTION
**Pull request by Mng-dev-ai**
This PR fix count of available convertors which are five not four as declared in docs.

### CODE REVIEW

File: docs/routing.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Route('/users/{username}', user)
 ```
 By default this will capture characters up to the end of the path or the next `/`.
 
-You can use convertors to modify what is captured. Four convertors are available:
+You can use convertors to modify what is captured. Five convertors are available:
```
Comment (aminalaee) [inline docs/routing.md]: Maybe remove that number completely so we won't face this again?
I mean the list is displayed right here, what's the importance of counting it?

**Review by aminalaee — COMMENTED** [review]
Thanks for the PR.

File: docs/routing.md (hunk 1 above)
Comment (Mng-dev-ai) [reply docs/routing.md]: Yes, That's would be much better.
Do you wanna me do it ?

Comment (Kludex) [reply docs/routing.md]: If you're willing, it would be cool. 😁

Comment (Mng-dev-ai) [reply docs/routing.md]: Done.

**Review by Kludex — APPROVED** [review]

**Pull request merged**